jobs.md

```diff
@@ -0,0 +1,33 @@
+# Jobs
+
+Come and work with us!
+
+## Upstream team
+
+Interested in working on open source and containers at a leading hyper-scale cloud? You can help build the future of Kubernetes at Microsoft!  
+
+### Engineering
+#### Secure supply chain
+- [Software Engineer II, Atlanta, remote](https://careers.microsoft.com/us/en/job/1210002/Software-Engineer-II)
+- [Senior Software Engineer, Atlanta, remote](https://careers.microsoft.com/us/en/job/1309571/Senior-Software-Engineer)
+
+#### Cloud native upstream
+- [Software Engineer II, Atlanta, remote](https://careers.microsoft.com/us/en/job/1224166/Software-Engineer-II)
+
+#### WASM
```

Comment:
These two can be dropped. I'll add when new openings are available.

Reply:
done